specs/altair/sync-protocol.md

```diff
@@ -130,10 +131,10 @@ def get_safety_threshold(store: LightClientStore):
 
 A light client maintains its state in a `store` object of type `LightClientStore` and receives `update` objects of type `LightClientUpdate`. Every `update` triggers `process_light_client_update(store, update, current_slot)` where `current_slot` is the current slot based on some local clock. `process_slot` is processed every time the current slot increments.
 
-### `process_slot`
+#### `process_slot`
 
 ```python
-def process_slot(store: LightClientStore, current_slot: Slot):
+def process_slot_for_light_client_store(store: LightClientStore, current_slot: Slot) -> None:
```

Comment:
Function name `process_slot` has been taken by core beacon chain spec. Here I just renamed it to something else. Feel free to propose a better name.

Reply:
Looks good to me!